prj.conf

```diff
@@ -20,12 +20,6 @@ CONFIG_USERSPACE=y
 
 CONFIG_MAIN_STACK_SIZE=32768
 
-# Enable cache due to vchan issue
```

Comment:
I believe that this patch will help you https://github.com/xen-troops/zephyr/pull/21/commits/7318d51b18badb65d0394107f49290a54302cf30

and you can leave these configs

Reply:
we don't use cache management here. It was added as a workaround